integration_test/third_party_apps_data/applications/solr/centos_rhel/install

```diff
@@ -0,0 +1,40 @@
+set -e
+
+sudo setenforce 0
+
+cat <<EOF | sudo tee /etc/selinux/config
+SELINUX=disabled
+SELINUXTYPE=targeted
+EOF
+
+sudo yum install -y \
+    java-1.8.0-openjdk java-1.8.0-openjdk-devel curl lsof
+
+curl -L -o \
+    solr-8.11.1.tgz \
+    https://downloads.apache.org/lucene/solr/8.11.1/solr-8.11.1.tgz
+
+tar -xzf \
+    solr-8.11.1.tgz \
+    solr-8.11.1/bin/install_solr_service.sh \
+    --strip-components=2
+
+sudo bash ./install_solr_service.sh solr-8.11.1.tgz -n
+
+sudo chown -R solr:solr /opt/solr*
+
+cat <<EOF | sudo tee /etc/default/solr.in.sh
```

Comment:
Can you edit the existing file in place (e.g. with `sed`) instead of copying the default values here? That will ensure that we are compatible with defaults even if the defaults change.